### DISCUSSION TITLE
Issue with running npm test main.test.js

### DISCUSSION THREAD

**Discussion by jackiebgregory** [nss-evening-cohort-14/evening-client-side#46] [Help Me!] [answered]
**Your Name:** Jackie

**Have you searched the other help tickets?** Yes. There were several with npm issues related to the js-practice assignments, and Dani shared a troubleshooting win, but none of it seemed to apply to my issue.

**Describe your issue:** When I worked on it last night, everything was working fine. I left CSCode open on my computer, and didn't touch it again until this evening. I answered another problem, ran npm test main.test.js, and it worked fine. Then, I answered another one, ran npm test main.test.js, and it just seemed to stop. It had been a little slow to run, so I waiting a bit longer, but still nothing. I checked other help tickets and discussions, but I'm not getting any error messages. I read through Dani's troubleshooting win on npm installs, and restarted my computer and tried those directions. Then tried running npm again, but still ... nothing.

**List any error messages that appear:** There are no error messages, it just seems to stop working.

**What have you googled?** 
https://stackoverflow.com/questions/54277015/npm-test-command-is-not-running
https://forum.freecodecamp.org/t/how-to-npm-test-js/55482
https://www.npmjs.com/package/test

**What have you tried? Tell us everything you have tried.**
I tried reading through all the help tickets, restarting the computer, checking out to main/master and reinstalling npm, then going back to attemp1 and trying it again.

Here's what I got when I did that - it just sort of seems to stop and not do anything else - there is no error message at the end.

➜  js-practice-2-jackiebgregory git:(attempt1) ✗ git checkout master
M       main.js
Switched to branch 'master'
Your branch is up to date with 'origin/master'.
➜  js-practice-2-jackiebgregory git:(master) ✗ sudo npm install
Password:
audited 734 packages in 9.334s

29 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

   ╭─────────────────────────────────────────────────────────────────╮
   │                                                                 │
   │      New patch version of npm available! 6.14.10 → 6.14.11      │
   │   Changelog: https://github.com/npm/cli/releases/tag/v6.14.11   │
   │                Run npm install -g npm to update!                │
   │                                                                 │
   ╰─────────────────────────────────────────────────────────────────╯

➜  js-practice-2-jackiebgregory git:(master) ✗ git checkout attempt1
M       main.js
Switched to branch 'attempt1'
➜  js-practice-2-jackiebgregory git:(attempt1) ✗ npm test main.test.js

> example@0.0.0 test /Users/cjgregory/workspace/foundations/exercises/js-practice-2-jackiebgregory
> jest "main.test.js"

 RUNS  ./main.test.js

**Comment by drteresavasquez** [comment]
Either check your formatting to make sure everything is correct, push it up to GitHub and let the test run there, or keep trying. If there are no errors your computer may just need some time to think ☺️

If you push it up to github and the tests pass, you will get a green check. If they fail, you will get a red x. That will let you know if you need to make some changes to your file.

**Comment by drteresavasquez** [comment]
@jackiebgregory let us know if anything you've done works. If not, we can continue the convo.

**Comment by jackiebgregory** [comment, marked as answer]
Follow up: After writing up this Help Ticket, I saved my worked, closed out of all programs, and shut the computer down for the night. When I tried it again the next day, it worked! I didn't do anything differently... maybe the computer just needed a break?

**Reply by drteresavasquez** [reply]
It's possible! A break is good for us all sometimes. Even machines 😃
